core/src/mindustry/ui/dialogs/CustomGameDialog.java

```diff
@@ -25,7 +25,7 @@ public CustomGameDialog(){
 
     void setup(){
         clearChildren();
-        add(titleTable);
+        add(titleTable).growX().row();
         row();
```

Comment:
You add a `.row()`, which is like a newline in tablespeak, yet directly under this is another `row()`. Methinks one of these `.row()`s is unnecessary, try without it and see.

`.growX()` is the important part, it makes the element expand to fit the width of its container.
```suggestion
```

Reply:
removing the `.row()` below it for consistency to keep it inline with the class found in arc.